Cpp_test/lib/ipc-handler.cpp

```diff
@@ -0,0 +1,128 @@
+#include "ipc-handler.h"
+#include <ctime>
+#include <iostream>
+
+
+void ipcHandler::select_options(IPCMode mode, int argc, char *argv[]){
+	this->info.mode = mode;
+  	int opt;
+	info.fflag = 0;
+	info.pflag = 0;
+	info.qflag = 0;
+	info.sflag = 0;
+  	info.hflag = 0;
```

Comment:
Seems like you could use bools for the flags as well if the only allowed values are 0 and 1. Then instead of adding these together and checking for != 1, you could just use the OR operator (||)

Reply:
Will do as you suggest